src/qualifier/QualifierValue.ts

```diff
@@ -1,5 +1,8 @@
 import {Expression} from "../values/expression/Expression";
 
+/**
+ * @memberOf SDK
```

Comment:
member of SDK?

Reply:
Yes - I've created a new namespace for things that don't belong under "system" or "core" - these are the SDK internals (Qualifier, QualifierValue, Action etc.)